os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -255,23 +281,24 @@ static inline void mpu_show_regioninfo(void)
 	/* save the current region before printing the information */
 	temp = getreg32(MPU_RNR);
 
-	lldbg("*****************************************************************************\n");
-	lldbg("*REGION_NO.\tBASE_ADDRESS\t    SIZE\t REG\t   \tSTATUS\tATTRIBUTES*\n");
-	lldbg("*****************************************************************************\n");
+	lldbg("********************************************************************************************************\n");
+	lldbg("%-12s\t%-12s\t%-12s\t%-12s\t%-12s\t%-12s\t%-12s\n", "REGION_NO.", "BASE_ADDRESS", "SIZE", "REG",  "STATUS", "ACCESS (P/U)", "EXECUTE");
+	lldbg("********************************************************************************************************\n");
 	for (idx = 0; idx < 8; idx++) {
 		putreg32(idx, MPU_RNR);
 		base = getreg32(MPU_RBAR);
 		size = getreg32(MPU_RASR);
 		attr = getreg32(MPU_RASR);
-		lldbg("%8d\t\t%8X\t%8X\t%8X\t%8X\t%8X\n",
+		lldbg("%-12d\t0x%-10X\t0x%-10X\t%0x%-10x\t%-12s\t%-12s\t%-12s\n",
 			(base & MPU_RBAR_REGION_MASK),
 			(base & MPU_RBAR_ADDR_MASK),
 			(size ? (1 << (((size & MPU_RASR_SIZE_MASK) >> MPU_RASR_SIZE_SHIFT) + 1)) : 0),
 			((size & MPU_RASR_SRD_MASK) >> MPU_RASR_SRD_SHIFT),
-			(size & MPU_RASR_ENABLE) ? 1 : 0,
-			((attr & MPU_RASR_ATTR_MASK) >> MPU_RASR_ATTR_SHIFT));
+			(size & MPU_RASR_ENABLE) ? "ENABLED" : "DISABLED",
+			mpu_get_access_type(attr),
```

Comment:
return value not handled for get_access_type? we can make function void in that case